src/index.ts

```diff
@@ -29,9 +27,25 @@ app.use(helmet());
 app.use(cors());
 app.use(express.json());
 app.get("/", function(_req, res) {
-  res.send(rhubarbVersion);
+  res.send("OK. " + rhubarbVersion);
 });
 
+// TODO, tidy up api documentation
```

Comment:
I'd recommend deleting these commented out things, if you need them again you can always check back in the commit history

Reply:
Done